packages/manager/src/features/VPCs/VPCDetail/VPCDetail.test.tsx

```diff
@@ -91,10 +95,7 @@ describe('VPC Detail Summary section', () => {
       }
     );
 
-    const loadingState = queryByTestId(loadingTestId);
-    if (loadingState) {
-      await waitForElementToBeRemoved(loadingState);
-    }
+    await waitForElementToBeRemoved(queryByTestId(loadingTestId));
 
     getAllByText('Subnets');
```

Comment:
Additional improvement: we should be using "getByText" over "getAllByText" and "expect" + "toBeVisible" whenever possile when checking for things in the DOM



```suggestion
    expect(getByText('Subnets')).toBeVisible();
```